### PR TITLE
matt submission

### PR DESCRIPTION
**Pull request by mattyding**
Final val loss: 4.12
- SwiGLU, 6 Transformer layers, hyperparam tuning, torch.compile()
- Training kept failing partway through :( so the best I got was this 40-min run.

As a note, the vanilla GELU performed much better than my SwiGLU implementation and didn't run into as much instability.

### CODE REVIEW

**Comment by nelson-liu** [comment]
thanks!
